### PR TITLE
Add support for mouse gestures

### PR DESCRIPTION
**Pull request by Download**
https://github.com/brutaldesign/swipebox/issues/222
Also bumped version number and added author and contributors sections to package.json

### CODE REVIEW

**Comment by Download** [comment]
Does this mean the pull request is rejected?

File: grunt/package.json [hunk 1]
```diff
@@ -1,11 +1,81 @@
...
     "type": "git",
     "url": "git://github.com/brutaldesign/swipebox.git"
   },
+  "author": {
+    "name": "Constantin Saguin",
+    "email": "hello@csag.co",
+    "url": "http://csag.co" 
+  },
+  "contributors": [
+    {
```
Comment (huysentruitw) [inline grunt/package.json]: Doesn't follow coding style

**Comment by huysentruitw** [comment]
Would be a nice to have feature, but I think your PR has some issues.

File: grunt/package.json (hunk 1 above)
Comment (Download) [inline grunt/package.json]: Can you elaborate? I don't see where my code deviates from the code style used by the rest of the file.

Comment (huysentruitw) [inline grunt/package.json]: Mind the opening brackets :)

"contributors": [{
  "name": "Dheera Venkatraman",
  "email": "dheera@dheera.net",
  "url": "http://dheera.net",
}, {
  // etc
}

**Comment by Download** [comment]
Ah thanks for clearing that up. I thought that maybe you were part of the brutaldesign team or something.

Yes I was hoping that @brutaldesign would explain why he closed it. Maybe there is something wrong with the commit as you suggest but it's better to hear from him so I know what he wants fixed.
Or maybe he just doesn't care for this feature. Which is also ok, but in that case it's not productive for me to start 'fixing' the pull request...

**Pull request closed without merging**